src/tests/ftest/util/mdtest_test_base.py

```diff
@@ -58,7 +63,24 @@ def setUp(self):
         self.processes = self.params.get("np", '/run/mdtest/client_processes/*')
         self.manager = self.params.get("manager", '/run/mdtest/*', "MPICH")
 
-    def execute_mdtest(self):
+        # Until DAOS-3320 is resolved run IOR for POSIX
+        # with single client node
+        if self.mdtest_cmd.api.value == "POSIX":
+            self.hostlist_clients = [self.hostlist_clients[0]]
+            self.hostfile_clients = write_host_file.write_host_file(
+                self.hostlist_clients, self.workdir,
+                self.hostfile_clients_slots)
+
+    def tearDown(self):
+        """Tear down each test case."""
+        try:
+            if self.dfuse is not None:
+                self.dfuse.stop()
```

Comment:
It would be a lot nicer if the Dfuse() object could call self.stop() from a __del__() handler, it would avoid this code and a couple of similar function that were added in the initial commit.